tests/tests.cc

```diff
@@ -29,14 +29,14 @@
 
 int main()
 {
-    // tests::connection_test st;
-    // st.run();
+    tests::connection_test st;
+    st.run();
 
     // tests::conversion_test ct;
     // ct.run();
 
-    tests::mission_test mt;
-    mt.run();
+    // tests::mission_test mt;
+    // mt.run();
```

Comment:
Is it really necessary to have all this commented code? Since it looks like it is not possible to run all 3 in parallel, can't we have either a test for each or an argument to set the right one?